### ISSUE TITLE
[Feature Request]: Allow user to change type of f1 score

### ISSUE THREAD

**Issue by Tyrest** [refuel-ai/autolabel#410]
**Is your feature request related to a problem? Please describe.**
It would be nice to be able to change the type of f1 score that is displayed as a metric.

**Describe the solution you'd like**
The user should be able to choose which f1 score they want to use in the config (maybe even a list of f1 scores?) and they should be calculated using `sklearn.metrics.f1_score`.

**Additional context**
See conversation [here](https://github.com/refuel-ai/autolabel/pull/405#discussion_r1245689196)

**Comment by nihit** [comment]
@Tyrest can this be closed out now?
